astrodendro/io/fits.py

```diff
@@ -35,10 +35,18 @@ def dendro_export_fits(d, filename):
     except AttributeError:
         primary_hdu = fits.PrimaryHDU()
 
+    col1 = fits.Column(name='min_npix', format='J',
+                       array=np.array([d.params['min_npix']]))
+    col2 = fits.Column(name='min_delta', format='E',
+                       array=np.array([d.params['min_delta']]))
+    col3 = fits.Column(name='min_value', format='E',
+                       array=np.array([d.params['min_value']]))
+
```

Comment:
Should these parameters be in the fits header instead of a new extension? That would seem more standard to me